lua/telescope/builtin/files.lua

```diff
@@ -78,7 +78,16 @@ files.live_grep = function(opts)
         search_list = filelist
       end
 
-      return flatten { vimgrep_arguments, prompt, search_list }
+      local extraArgs = {}
+      local smart_mask = utils.get_default(opts.smart_mask, conf.smart_mask)
+      local filetype = opts.current_filetype or ''
```

Comment:
yep just move local filetype = vim.bo.filetype outside of inner function `live_grepper`

Reply:
Ah, that was it. _Outside_ of inner function did the trick. Thanks!